autoload/health/floaterm.vim

```diff
@@ -8,30 +8,30 @@
 let s:home = fnamemodify(resolve(expand('<sfile>:p')), ':h:h:h')
 
 function! s:check_common() abort
-  call health#start('common')
-  call health#info('Platform: ' . s:get_platform_info())
-  call health#info('Nvim: ' . s:get_nvim_info())
-  call health#info('Plugin: ' . s:get_plugin_info())
+  call vim.health.start('common')
```

Comment:
Thanks for the PR but this seems not valid vimscript. AFAIK, there isn't global variables like `vim`.